cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -1197,6 +1214,7 @@ func init() {
 	endDevicesCommand.AddCommand(endDevicesClaimCommand)
 	endDevicesCommand.AddCommand(endDevicesListQRCodeFormatsCommand)
 	endDevicesGenerateQRCommand.Flags().AddFlagSet(endDeviceIDFlags())
+	endDevicesGenerateQRCommand.Flags().AddFlagSet(formatterParameterFlags("formatters"))
```

Comment:
Why are you adding payload formatter flags to the QR code generation command?

Reply:
Ahh, indeed that does not make sense.